src/test/java/com/google/javascript/clutz/refer_from_default_ns.js

```diff
@@ -0,0 +1,10 @@
+goog.provide('fn');
+goog.provide('fn.String');
+
+/** @return {fn.String} */
```

Comment:
It does - the issue is that fn is essentially a `var`, so the namespace while visiting `fn` is `''`.